setup.py

```diff
@@ -1,20 +1,38 @@
-import setuptools
+from setuptools import setup, find_packages
 
-with open("README.md", "r") as fh:
-    long_description = fh.read()
+with open("README.md") as f:
```

Comment:
in general, I would appreciate not removing `style` data. imo having the file direction specified is good or at least it does no harm.